libshaderc/include/shaderc/shaderc.h

```diff
@@ -86,6 +86,93 @@ typedef enum {
   shaderc_optimization_level_size,  // optimize towards reducing code size
 } shaderc_optimization_level;
 
+// Resource limits.
+typedef enum {
```

Comment:
It is unfortunate to have to specify everything twice, once in the .inl file and once here, but I understand the desire to keep the header as clean as possible.